apps/web/src/components/layouts/Cards/Status.tsx

```diff
@@ -1,11 +1,11 @@
+import type { FursonaStatus } from "@/types/characters"
```

Comment:
Since we're targeting to a wide range of audience, consider renaming the "fursona" imports?

```suggestion
import type { CharacterStatus } from "@/types/characters"
```